experimental/dds/tree2/src/codec/parameterized-dds.md

```diff
@@ -0,0 +1,252 @@
+# Parameterized DDSes
+
+This design document centers around the problem of providing better support for "parameterized DDSes,"
+aka a DDS which is configurable in some way by an application author.
+The current APIs are generally sufficient for DDSes whose only parameters affect runtime behavior in a way that doesn't have compatibility issues.
```

Comment:
I'd kind of disagree with this statement.  Parameterizing compatibility-affecting qualities is definitely harder, but we don't currently have a good parameterization story even for runtime-only behaviors IMO.